content/blog/get-started-with-prometheus-and-grafana-on-docker-with-hpe-storage-array-exporter.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Get started with Prometheus and Grafana on Docker with HPE Storage Array
+  Exporter
+date: 2021-12-07T16:00:00.000Z
+author: Michael Mattsson
+authorimage: /img/portrait-192.jpg
+tags:
+  - HPE Alletra
+  - HPE Nimble Storage
+  - Kubernetes
+  - Prometheus
+---
+With the recently released HPE Storage Array Exporter for Prometheus and HPE CSI Info Metrics Provider for Prometheus, it's a good time to familiarize ourselves with the cloud native technologies involved and get some first hand experience.
+
+Prometheus is a time-series database that also provides monitoring and alerting. It's a [CNCF graduated project](https://www.cncf.io/projects/). Grafana is a web-based visualization tool that uses time-series data to create beautiful graphs and elements to present data in views referred to as dashboards. Prometheus scrapes an HTTP endpoint of a target periodically to consume an assortment of metric types and and metadata. The target is usually referred to as an exporter and the data being scraped is the current state of one or many exporter data points.
+
+![Prometheus Overview](/img/prometheus-1.0.0-beta-revwip.png)
```

Comment:
Hi, Michael. For instances where we may use a "blog preview" screen to promote the blog post, the preview picks up the first image in the post, which would be this image for this post. I was wondering if you could modify this image slightly because of this. Would you mind taking out the asterisk after "OVERVIEW" in the title? And, at the very bottom, take out the "* =" prior to the URL, and move the URL over a smidge to the right, please, to give the element a bit more "breathing room". [these are things Brand would request... please don't shoot the messenger)

Reply:
This slide is a placeholder. I'm working on a HPE branded version.